climakitae/util/warming_levels.py

```diff
@@ -53,8 +53,17 @@ def _calculate_warming_level(warming_data, gwl_times, level, months, window):
 
     # Check that there exist simulations that reached this warming level before cleaning. Otherwise, don't modify anything.
     if not (warming_data.centered_year.isnull()).all():
-        # Removing simulations where this warming level is not crossed. (centered_year)
-        warming_data = warming_data.sel(all_sims=~warming_data.centered_year.isnull())
+        # Removing simulations where this warming level is not crossed (centered_year), or if the simulation would not return a full set of data for the given warming level (warming level window lower year is < 1980, or warming level window upper year > 2100)
+
+        import pdb; pdb.set_trace()
+        is_null_sim_mask = warming_data.centered_year.isnull()
+        sims_to_drop = warming_data.all_sims.where(is_null_sim_mask, drop=True)
+
+        if len(sims_to_drop.all_sims.values) > 1:
+            print(f"These are the following simulations and their given SSPs to be dropped since they do not reach return a full set of WL data for this given WL: {sims_to_drop.all_sims.values}")
```

Comment:
Add WL number in print statement, in case this call creates a DataArray across multiple warming levels